src/models/Card/Card.ts

```diff
@@ -18,7 +18,11 @@ class Card extends Utils {
   private static DEFAULT_LANGUAGE: string = 'en_GB';
   private static DISABLED_ATTRIBUTE: string = 'disabled';
   private static ERROR_CLASS: string = 'error';
-  private static NOT_FLIPPED_CARDS: string[] = [CARD_TYPES.AMEX];
+  private static NOT_FLIPPED_CARDS: string[] = [CARD_TYPES.AMEX, CARD_TYPES.PIBA];
```

Comment:
info: there is no need to flip card if PremierInn card is indicated

Reply:
Agreed :) Nice